scilpy/segment/streamlines.py

```diff
@@ -66,19 +58,22 @@ def filter_grid_roi(sft, mask, filter_type, is_not):
             line_based_indices = np.union1d(line_based_indices_1,
                                             line_based_indices_2)
 
```

Comment:
if:
elif:

There is no else? normal?

Reply:
I will put an else, but leave a comment with the condition so it is clear which one it is